code/modules/clothing/under/excelsior.dm

```diff
@@ -1,10 +1,18 @@
 /obj/item/clothing/under/excelsior
 	name = "random excelsior jumpsuit"
-	desc = "Excelsior jumpsuit designed to boost morale and spread the revolution"
+	desc = "Reject decadent capitalist 'fashion', embrace the liberation of true revolutionary style"
 	icon_state = "excelsior_white"
 	item_state = "bl_suit"
 	has_sensor = 0
 	spawn_blacklisted = TRUE
+	armor = list(
+		melee = 3,
+		bullet = 4,
```

Comment:
giving it bullet armor this high would make it the single best uniform in the game. the only other armored uniforms, Rockstar Outfit and OneStar Jumpsuit have 1 and 2 bullet armor respectively

Reply:
yeah I was considering the effects of that...it would be best to drop it to 2 maybe, I just thought it'd be a good way to incentivise people using the excel jumpsuits.